models/user/user.go

```diff
@@ -728,6 +728,7 @@ func createUser(ctx context.Context, u *User, meta *Meta, createdByAdmin bool, o
 
 	u.LowerName = strings.ToLower(u.Name)
 	u.AvatarEmail = u.Email
+	u.Email = strings.ToLower(u.Email)
```

Comment:
No, it's not right.

Reply:
Actually I'm also not sure if it's right. I noticed that we always change the email to lowercase when updating a user. So I think maybe we can also change it to lowercase when creating the user.

https://github.com/go-gitea/gitea/blob/3c00e891296caf3ec246e3e9b4800e011b6945c1/models/user/user.go#L179-L185

Reply:
I believe such trick is not needed anymore, because the email is mainly managed by the email table which has "lower_email" field.

I would like to revert this unnecessary change.

Reply:
Yes. It looks unnecessary. Reverted in 4a03afc086db511fb4f230b250824d3414bc9306